aws/ec2/install

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Install dependencies with Yum
+sudo yum -y update
+sudo yum -y install git jq
+
+# Install RVM
+gpg --batch --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
```

Comment:
This line is not importing the keys properly. If I run that same line twice, everything works as expected. Trying to figure out what I am supposed to do before running this line...